docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Device_operations/Overview.md

```diff
@@ -2,4 +2,4 @@
 
 ## How synchronization works
 
-Coiote DM provides zero-touch provisioning for synchronized devices from Azure IoT Central. This means that device entities are automatically created within Coiote DM upon synchronization with the Azure IoT Central and this is repeated periodically for any new devices that appear. Therefore, after one successful synchronization, you can be sure that any devices that have been added to Azure IoT Central at a later time will also be migrated to Coiote DM.             
+Coiote DM provides zero-touch provisioning for devices synchronized with the Azure IoT Central. This means any devices imported using a CSV file are automatically created within Coiote DM and the Azure IoT Central and from this time on are synchronized periodically. Therefore, after any successful synchronization, you can be sure that any devices that have been imported via Coiote DM will be presented in their actual state within the Azure IoT Central.                      
```

Comment:
I have a question about the definite article: should we write "the Azure IoT Central" or just "Azure IoT Central"? I'd say that we need **the** when there's another word added. For example: "the Azure IoT Central application" (or "the Coiote DM platform), but if it's just the name, then no article is needed: Azure IoT Central, Coiote DM. I'm not sure how to ask google about this grammar rule, but I think it exists :)

Reply:
Oh, Coiote DM doesn't have any articles, so the question is only about Azure IoT Central

Reply:
"we need the when there's another word added"
agree, it's exactly the case.
I'll correct it